install/sensors.csv

```diff
@@ -832,6 +832,7 @@ Nanometrics,Trillium 240,161,WEL,11,0,,0,0,0,0,0,0,2014-07-25T00:05:00Z,2014-11-
 Nanometrics,Trillium 360,970,BKZ,10,0,,0,0,0,0,0,0,2020-07-02T23:35:00Z,9999-01-01T00:00:00Z
 Nanometrics,Trillium Borehole T120-BH1,1073,URZ,10,216,,0,50,0,0,0,0,2021-05-26T23:47:00Z,9999-01-01T00:00:00Z
 Nanometrics,Trillium Borehole T120-BH1,1074,RPZ,10,182,,0,50,0,0,0,0,2021-06-24T05:30:00Z,9999-01-01T00:00:00Z
+Nanometrics,Trillium Borehole T120-BH2,003614,COVZ,11,0,,0,59,0,0,0,0,2024-09-25T02:17:00Z,9999-01-01T00:00:00Z
```

Comment:
no need to update the azimuth?

Reply:
@mnaguit the orientation will need to be changed since this is a new borehole install. We'll have to update in again once the sensor has been in place long enough to get an orientation. In the meantime I think it's tidier to set it to zero, but I can change it to something else if you think that's a good idea.

Reply:
Sounds good Sir @Thomas-Benson 😃